### ISSUE TITLE
Create statistics index

### ISSUE THREAD

**Issue by y0sh188** [dtag-dev-sec/PEBA#55]
Hi,

I'd like to know how to create statistics index in elasticsearch.
I accessed public get endpoint 'alert/getStats?' on my peba server,  I got this error message.

`[2019-08-27 07:24:15,157] ERROR in tpotstats: ElasticSearch error: TransportError(404, 'index_not_found_exception', 'no such index')`

Thanks,

**Comment by vorband** [comment]
did you setup the indices? The script is located at ./misc/setup-es-indices.py

**Comment by y0sh188** [comment]
There is no "statistic index" in "setup-es-indices.py"

**Comment by vorband** [comment]
that's correct. Currently the statistics stuff is not in the public repo, hence you cannot use this endpoint,. There is a job script which needs to be added. I will look into this later.

**Comment by y0sh188** [comment]
I see. Is it possible to share a job script and index script?

**Comment by vorband** [comment]
I will check this. Not sure I can.
